pkgs/applications/misc/yubioath-flutter/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, flutter
+, python3
+, fetchFromGitHub
+, stdenv
+, pcre2
+}:
+
+let
+  vendorHashes = {
+    x86_64-linux = "sha256:0bwrybwaq9q16dwr56590k8j66qfckrjihsqj9brwjvlqc1mc207";
+  };
+
+  version = "6.0.2";
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubioath-flutter";
+    rev = "${version}";
+    sha256 = "13nh5qpq02c6azfdh4cbzhlrq0hs9is45q5z5cnxg84hrx26hd4k";
+  };
+
+  helper = python3.pkgs.callPackage ./helper.nix { inherit src version; };
+in
+flutter.mkFlutterApp rec {
+  pname = "yubioath-flutter";
+  inherit src version;
+
+  passthru.helper = helper;
+
+  vendorHash = vendorHashes."${stdenv.system}";
+
+  prePatch = ''
+    substituteInPlace linux/CMakeLists.txt \
+      --replace "../build/linux/helper" "${helper}/libexec/helper"
+  '';
+
+  preInstall = ''
+    # Make sure we have permission to delete things CMake has copied in to our build directory from elsewhere.
+    chmod -R +w build/
+  '';
+  postInstall = ''
+    # Swap the authenticator-helper symlink with the correct symlink.
+    rm "$out/app/helper/authenticator-helper"
+    ln -s "${helper}/bin/authenticator-helper" "$out/app/helper/authenticator-helper"
+  '';
+
+  buildInputs = [
+    pcre2
+  ];
+
+  meta = with lib; {
+    description = "Yubico Authenticator for Desktop";
+    homepage = "https://github.com/Yubico/yubioath-flutter";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ lukegb ];
+    platforms = builtins.attrNames vendorHashes;
```

Comment:
```suggestion
    platforms = lib.attrNames vendorHashes;
```

Reply:
This is not strictly necessary (or useful: the lib name is IMO only useful if it does something other than alias). nixpkgs (including lib/!) already includes many instances of just using builtins.attrNames; the lib alias is now really only useful for doing bulk `inherit (lib) ..... attrNames;`.